src/css/custom.css

```diff
@@ -534,6 +558,49 @@ body {
   }
 }
 
+/* Hide table of contents sidebar */
+.col.col--3 {
+  display: none !important;
+}
+
+.table-of-contents,
+.theme-doc-toc-desktop {
+  display: none !important;
+}
+
+/* Force center all documentation content */
+.col:not(.col--3) {
+  max-width: 1000px !important;
+  margin-left: auto !important;
+  margin-right: auto !important;
+}
+
+/* Force section overview to center */
+.generatedIndexPage_vN6x {
+  max-width: 980px;
+  margin: 0 auto;
+  float: none;
+}
+
+.generatedIndexPage_vN6x .row .col--6 {
+  flex: 0 0 calc(50% - 0.75rem) !important;
+  max-width: calc(50% - 0.75rem) !important;
+}
+
+.generatedIndexPage_vN6x .row {
```

Comment:
is this className consistent? maybe there's a better way to do this that's more reliable, the vN6x looks generated